opentelemetry-stackdriver/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v0.13.0
+
+### Changed
+
+- Exclude default `tonic-build` features #635
```

Comment:
```suggestion
- Send export errors to global error handler (#705)
- Return `impl Future` to avoid spawning inside library (#703)
- Implement builder API to simplify configuration (#702)
- Use TLS configuration provided by tonic (#702)
- Optionally send events to Cloud Logging (#702)
- Exclude default `tonic-build` features #635
```